src/TextareaAutosize.js

```diff
@@ -87,6 +93,13 @@ export default class TextareaAutosize extends React.Component {
     // Re-render with the new content then recalculate the height as required.
     this.onNextFrameActionId = onNextFrame(this._resizeComponent);
   }
+  
+  componentWillUpdate(nextProps, nextState) {
```

Comment:
@Andreyco regarding the infinite loop due to parent's change to textarea height, I agree.